### PR TITLE
feat: add getting started topics

### PR DESCRIPTION
**Pull request by fzn0x**
Follow up https://github.com/sequelize/website/pull/67#issuecomment-1088390814 comment

### CODE REVIEW

File: src/components/homepage-features.tsx [hunk 1]
```diff
@@ -103,7 +103,32 @@ export default function HomepageFeatures(): JSX.Element {
...
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/other-data-types/#json-sqlite-mysql-mariadb-and-postgresql-only"
+          > JSON querying
+          </Link>,
+          <Link
+            className="topics"
+            to="/docs/v6/other-topics/hooks/"
+          > lifecycle events (hooks)
+          </Link>
```
Comment (ephys) [inline src/components/homepage-features.tsx:130]:
```suggestion
          </Link>,
```

**Review by ephys — CHANGES_REQUESTED** [review]
This project uses yarn, not npm. Please remove the package-lock file
There is also no good reason for having updated the lockfile, please reset yarn.lock to its previous state

File: src/components/homepage-features.tsx (hunk 1 above)
Comment (fzn0x) [reply src/components/homepage-features.tsx:130]: By the way, I'm calling yarn and the lock file is updated

Comment (fzn0x) [reply src/components/homepage-features.tsx:130]: Sure I will rever

Comment (ephys) [reply src/components/homepage-features.tsx:130]: > By the way, I'm calling `yarn` command and the lock file is updated

It shouldn't, yarn has no reason to update the lockfile if package.json has not changed. It's not happening on my end

Comment (fzn0x) [reply src/components/homepage-features.tsx:130]: > > By the way, I'm calling `yarn` command and the lock file is updated
> 
> It shouldn't, yarn has no reason to update the lockfile if package.json has not changed. It's not happening on my end

It's happen to me so I assume my machine broke xD

Comment (fzn0x) [reply src/components/homepage-features.tsx:130]: You can take a look to the changes there is no package,json changes too

Comment (fzn0x) [reply src/components/homepage-features.tsx]: Or..  it's happen because I'm calling the `npm install` command

Comment (ephys) [reply src/components/homepage-features.tsx]: That's probably why
Revert it, delete your node_modules, then call `yarn` and you should end up with the proper state :)

Comment (fzn0x) [reply src/components/homepage-features.tsx]: > That's probably why Revert it, delete your node_modules, then call `yarn` and you should end up with the proper state :)

Cool, thanks

**Comment by fzn0x** [comment]
The branch seems fine before draft deployment, I was suspecting the failing netlify build it causes by different repositories.

**Comment by ephys** [comment]
If that were the case the other PRs would be failing too. I think it's caused by your lockfile changes. Try resetting the lockfile to its current state in the repository

**Comment by WikiRik** [comment]
I think we only had PRs from non-forked branches before, so it could very well be the case that we don't allow the secrets of Netlify to be used for forked repositories. Not sure how to easily fix that though

**Comment by ephys** [comment]
Ah yes, that's a good point. https://github.com/sequelize/website/pull/102 is failing too

**Comment by fzn0x** [comment]
Maybe we ready to merge it, I guess? @WikiRik  @ephys

**Comment by ephys** [comment]
You still need to reset the changes made to yarn.lock

**Comment by fzn0x** [comment]
> You still need to reset the changes made to yarn.lock

I already did the same steps as https://github.com/sequelize/website/pull/101#discussion_r851654186 your comment, and this is the commit https://github.com/sequelize/website/pull/101/commits/d55359039d16e56d3838d286ed06fd39807ea24c

**Comment by WikiRik** [comment]
Yeah, yarn.lock is updated due to not using `yarn install --frozen-lockfile` the first time. But since we have renovate dealing with dependency management we should revert the yarn.lock to its state on main before we merge this

**Comment by fzn0x** [comment]
I think the latest merge branch was already updating the `yarn.lock`, we don't really need to revert it, and the netlify build is failing because of the different repository.

Do we ready to merge this now? so I can focus to another PR.

File: src/components/homepage-features.module.scss [hunk 2]
```diff
@@ -33,6 +33,10 @@
   margin-bottom: 0;
 }
 
+.topics {
```
Comment (WikiRik) [inline src/components/homepage-features.module.scss:36]: image
This doesn't seem to work locally (using Chrome 100)

**Review by WikiRik — CHANGES_REQUESTED** [review]

**Comment by fzn0x** [comment]
Already merge branches with latest dependabot update :D

**Comment by fzn0x** [comment]
I will re-create the PR inside sequelize-website repo. Just to make sure to pass the build.

**Pull request closed without merging**